package.json

```diff
@@ -72,8 +72,8 @@
     "typescript": "^4.0.2"
   },
   "peerDependencies": {
-    "react": "~16",
-    "react-dom": "~16",
+    "react": "~17",
```

Comment:
We should do something like this: `"react": "^16.8.0 || ^17.0.0"`, the way you're doing affects users running React 16, same with react-dom `"react-dom": "^16.8.0 || ^17.0.0"`

Reply:
```suggestion
    "react": "~17 || ^16.8.0",
```